### ISSUE TITLE
TrueOS 18.03 /usr/sbin/freebsd-update references rc.conf.pcbsd

### ISSUE THREAD

**Issue by mrT55** [trueos/trueos#133]
/usr/sbin/freebsd-update

```
# Set to force updates on specific files that we don't want changed
FORCEUPDATES="/etc/rc.conf.pcbsd /etc/rc /boot/loader.conf.pcbsd"
```
I discovered this while I was searching for why iocage fetch (py36-iocage-0.9.10)
was fetching from http://download.freebsd.org/ftp/releases/amd64/11.2-RELEASE
instead of a one of the trueos.org IP addresses.

Note: iocage Version 0.9.10 2017/12/22 will automatically call freebsd-update
when running iocage fetch.

**Comment by kmoore134** [comment]
18.03 is deprecated, this is fixed in 18.06 and later.

**Comment by mrT55** [comment]
OK, so where is the release/stable download for 18.06?
    I have checked:
```
         https://pkg.trueos.org/iso/18.06/
          (these are all for server ONLY)
            TrueOS-Stable-x64-18.06.iso                        03-Aug-2018 21:47           864722944
            TrueOS-Stable-x64-18.06.iso.md5                    09-Aug-2018 14:36                  33
            TrueOS-Stable-x64-18.06.iso.sha256                 09-Aug-2018 14:36                  65
.
         https://pkg.trueos.org/iso/

         https://trueos.org/downloads/  points to:  pkg.trueos.org/iso/18.06
                                           and:     download.trueos.org/master/amd64/
                                                        (which has only ver-18.03)
         also checked: download.trueos.org

         http://project-trident.org/download
            This says: TrueOS (18.06+)
                       Coming Soon: Instructions for enabling the package repository for Project Trident.

         www.trueos.org/blog/trueosdownstream/
            Nothing new.
```
